src/Credentials/CredentialProvider.php

```diff
@@ -102,7 +102,13 @@ public static function defaultProvider(array $config = [])
             );
```

Comment:
Please add a changelog file in the .changes/nextrelease folder as described at the bottom of [this page](https://github.com/aws/aws-sdk-php/blob/master/CONTRIBUTING.md)

Reply:
done, thx